roundedimageview/src/main/java/com/makeramen/roundedimageview/RoundedTransformationBuilder.java

```diff
@@ -16,30 +16,34 @@
 
 package com.makeramen.roundedimageview;
 
+import android.content.Context;
 import android.content.res.ColorStateList;
 import android.content.res.Resources;
 import android.graphics.Bitmap;
 import android.util.DisplayMetrics;
 import android.util.TypedValue;
 import android.widget.ImageView;
+
+import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
+import com.bumptech.glide.load.resource.bitmap.BitmapTransformation;
```

Comment:
If both picasso and glide imports are in the same file, that means a user has to have glide and picasso at the same time right? Should we have 2 different transformation types instead? With shared logic in an abstract class perhaps?